tests/lean/rewrite.lean

```diff
@@ -34,7 +34,7 @@ subst y;
 exact rfl
 
 theorem ex4 (x y z) (h₁ : 0 + x = y) (h₂ : 0 + y = z) : x = z := by
-rewrite [appendAssoc] at *; -- Error
+rewrite [appendAssoc,] at *; -- Error
```

Comment:
Perhaps best to assemble all of these tests into a single file, so it's easier to understand? e.g. `test/lean/trailingComma`.

It would also be good to exercise it further. Have the tests demonstrate that whitespace (including new lines) before or after the comma has no effect. 

Possibly also a test that checks for the appropriate error message on a double trailing comma?

Reply:
I moved all of them into one file